mirrord/layer/src/macros.rs

```diff
@@ -124,7 +124,7 @@ macro_rules! graceful_exit {
             nix::sys::signal::Signal::SIGTERM,
         )
         .expect("unable to graceful exit");
-        panic!()
+        std::process::exit(0)
```

Comment:
This is called when we can't kill the current process. Shouldn't we return a non-zero exit code?